nixos/modules/system/boot/systemd.nix

```diff
@@ -497,11 +498,14 @@ in
       '';
     };
 
-    systemd.generator-packages = mkOption {
```

Comment:
We would need a changelog entry for that, explaining people should use `systemd.packages` instead.

Reply:
Won't https://github.com/NixOS/nixpkgs/pull/65376/commits/717b8b3219e0a207e16e28f828f88060b0477d0b#diff-94706aabdc6412bb34118f0202004761R224 this be enough?

Reply:
Yeah, it could work. I had to stare at the diff while rebasing the changes from https://github.com/NixOS/nixpkgs/pull/66856 until I figured it out.

Maybe a single line in "notable changes", explaining the new behaviour of `systemd.packages`, that it got support for system-generators (or basically all `$package/lib/systemd/system-*`) probably shouldn't hurt either.

Reply:
Done in 9b30cf0cb4c0722ea1f2b85e5cb5b334d8516dc3

Reply:
Thanks!